CMakeLists.txt

```diff
@@ -4,20 +4,12 @@ project(demonstrator)
 # Print the value of CMAKE_SOURCE_DIR for debugging
 message(STATUS "CMAKE_SOURCE_DIR: ${CMAKE_SOURCE_DIR}")
 
-include(${CMAKE_BINARY_DIR}/conan_toolchain.cmake)
```

Comment:
Do not include toolchains, toolchains are passed as arguments (or automatically by presets)